sidebars.js

```diff
@@ -126,6 +126,18 @@ module.exports = {
           type: 'autogenerated',
           dirName: '001_develop/03_client-capabilities'
         },
+        {
```

Comment:
I don't think you should need to change sidebar as we set the section as autogenerated above. Trying to keep it clean so there are less places to edit if we need to re-order things. Does it not show up if you remove these changes?

Reply:
Re-checking this.. but I doubt it. Due the fact that we have the "api docs" folder there 

But will check again, this is what we had before, so I assume it will have the same behaviours

Reply:
@jay-taylerson updated here https://github.com/genesiscommunitysuccess/docs/pull/1857/commits/c0cdafb8dad4fe834adaaee5845ba6d7e3dea1c4

Got it working in a way that should be "ok" for now (assuming we're going with a long readme + with everything in there/less pages).